docs/README.md

```diff
@@ -173,6 +173,11 @@ publish whenever a pull request is merged.
 <p>Follow <a href="https://semver.org" target="_blank">semver</a> when choosing versions.</p>
 <code>npm run release-the-kraken</code>
 </details>
+<details>
+<summary>Publishing a failed release to NPM</summary>
+<p>In some cases, the automatic release may successfully bump the version and add a [changelog](https://atlantis.getjobber.com/packages-components-changelog) but fail to publish to NPM. If this happens, run the code below to send unpublished versions to NPM.</p>
+<code>npm run release:unpublished-package</code>
```

Comment:
Can anyone run this command, or only Atlantis npm maintainers?

Reply:
Anyone can technically run it. But only the ones that are authenticated in NPM can successfully release it

Reply:
If that group consists of only these people it might be good to clarify so someone doesn't repeatedly fail - and so someone can quickly see who has those permissions.
<img width="375" alt="image" src="https://user-images.githubusercontent.com/39704901/166835154-62eab1ea-6ab9-4771-a1b0-69f4e91b84f8.png">

Reply:
Also need to get Rodrigo and Michael added to that list 😄

Reply:
Fair point! I will add a disclaimer. I wonder if I have the power to add people

Reply:
I think maybe only @lmorgan29 has the power on our team...